File: Python/conversion_du_temps.py
temps=int(input("Entrez le temps en secondes: "))
jour= temps // 86400
reste= temps % 86400
heure= reste // 3600
reste= reste % 3600
minute= reste // 60
seconde= reste % 60

print("Le temps servir est le suivant: ")
if jour>0 :
    if jour>1 :
        print(f"{jour} jours")
    else :
        print(f"{jour} jour")

if heure>1 :
    print(f"{heure} heures")
else :
    print(f"{heure} heure")

if minute>1 :
    print(f"{minute} minutes")
else :
    print(f"{minute} minute")

if seconde>1 :
    print(f"{seconde} secondes")
else :
    print(f"{seconde} seconde")

File: JavaScript/conversion_du_temps.js
const temps=prompt('entrez le temps en seconde: ')*1
const jour=(temps/86400)
let rest =(temps % 86400)
const heure=(rest/3600)
let reste =(rest % 3600)
const minute=(reste/60)
const seconde=(reste % 60)
console.log('le temps saisie est le suivant: ')
if (jour>0){
    if (jour>1){
        console.log(Math.floor(jour),' jours')
    }else{
        console.log(Math.floor(jour),' jour')
    }
}
if (heure > 1){
    console.log(Math.floor(heure),' heures')
}else{
    console.log(Math.floor(heure),' heure')
}
if (minute > 1){
    console.log(Math.floor(minute),' minutes')
}else{
    console.log(Math.floor(minute),' minute')
}
if (seconde > 1){
    console.log(Math.floor(seconde),' secondes')
}else{
    console.log(Math.floor(seconde),' seconde')
}